### PR TITLE
Add a test for the roc-ifiation and switch it to full regexp.

### PR DESCRIPTION
**Pull request by etiennesegonzac**

### CODE REVIEW

File: poplar.js [hunk 1]
```diff
@@ -55,14 +55,14 @@ poplar.parse = function(html) {
...
-    .replace(/\n/g, '')
+    .replace(/\s*\n\s*/g, '')
     .replace(/  +/g, '')
 
     // attribute bindings
     .replace(regex.attrs, function(match, start, content, end) {
       var dynamic = [];
       var simple = content.replace(regex.attr, function(match, name, prop) {
-        dynamic.push(`${name}=${encodeURIComponent(prop)}`);
+        dynamic.push(name + '=' + encodeURIComponent(prop));
```
Comment (wilsonpage) [inline poplar.js:65]: Is this faster?

File: poplar.js [hunk 2]
```diff
@@ -55,14 +55,14 @@ poplar.parse = function(html) {
   debug('parse', html);
 
   var formatted = html
-    .replace(/\n/g, '')
+    .replace(/\s*\n\s*/g, '')
```
Comment (wilsonpage) [inline poplar.js:58]: What does this regex do?

File: poplar.js (hunk 1 above)
Comment (wilsonpage) [inline poplar.js:65]: Or just cross platform I'm guessing

Comment (etiennesegonzac) [inline poplar.js:65]: yeah just a cross platform change so that I could run the bench on all browsers

File: poplar.js (hunk 2 above)
Comment (etiennesegonzac) [inline poplar.js:58]: removes all the line returns _and_ potential spaces just before/after, like trailing spaces for example

**Pull request merged**